src/auth/server_impl/store/redis_kv.rs

```diff
@@ -24,7 +24,7 @@ impl KVStore for RedisKV {
 
     async fn write(&self, key: String, value: String) -> Result<(), KVError> {
         let mut con = self.0.get().await?;
-        con.hset(key, AUTH_FIELD, value).await?;
+        con.hset::<_, _, _, ()>(key, AUTH_FIELD, value).await?;
```

Comment:
This to fix the clippy error. Related rust lang issue: https://github.com/rust-lang/rust/issues/123748